config/v1/0000_10_config-operator_01_oauth.crd.yaml

```diff
@@ -642,9 +642,12 @@ spec:
                     the maximum amount of time that can occur between consecutive
                     uses of the token. Tokens become invalid if they are not used
                     within this temporal window. The user will need to acquire a new
-                    token to regain access once a token times out. If this value is
-                    not set, then tokens are valid until their expiry. Takes valid
-                    time duration string such as "5m", "1.5h" or "2h45m".
+                    token to regain access once a token times out. Takes valid time
+                    duration string such as "5m", "1.5h" or "2h45m". The minimum allowed
+                    value for duration is 300s (5 minutes). If the timeout is also
+                    configured per client, then that value takes precedence. If the
```

Comment:
be a little more precise here by dropping "also": client can define a timeout while this one here is nil. Still we want to apply the client one.

Reply:
@sttts  The intention was to let user know which timeout takes precedence if both are set. If only one of them is set, I thought there would be no ambiguity on which value is used. So, do you think we need to be more explicit even in the case where only one timeout is set?

Reply:
> If the timeout value is not specified and the client does not override the value,	then tokens are valid until their lifetime.

This would explain the behavior if none of them is set.

Reply:
Maybe, I am finding it little difficult to explain how two different API fields work together here :)

Reply:
I think your text without the "also" is just fine.

Reply:
Agreed. Addressed the nit.